xc7/requirements.txt

```diff
@@ -1,3 +1,13 @@
 python-constraint
 git+https://github.com/symbiflow/fasm
 git+https://github.com/symbiflow/xc-fasm
+
```

Comment:
Can we add this to `xc7/litex_demo/requirements.txt` rather than including it directly in the top level xc7 requirements?

IE These are only needed for the LiteX demo.

Reply:
fixed